pgcli/main.py

```diff
@@ -469,6 +469,9 @@ def cli(database, user, host, port, prompt_passwd, never_prompt, dbname,
         print('Version:', __version__)
         sys.exit(0)
 
+    if not os.path.exists(os.path.expanduser('~/.config/pgcli')):
+        os.mkdir(os.path.expanduser('~/.config/pgcli'))
```

Comment:
@amjith @inkn : Ok I got it. There should be `os.makedirs` instead of `os.mkdir` here. Travis apparently does not have `~/.config/` dir available.

P.S. @amjith: I'll make sure that when the behavioral tests fail, the output is clearly seen (not just "hey I didn't see what I expected").